DeepSpeech.py

```diff
@@ -253,11 +252,13 @@ def calculate_mean_edit_distance_and_loss(iterator, dropout, reuse):
 # we will use the Adam method for optimization (http://arxiv.org/abs/1412.6980),
 # because, generally, it requires less fine-tuning.
 def create_optimizer():
-    optimizer = tfv1.train.AdamOptimizer(learning_rate=FLAGS.learning_rate,
+    learning_rate_var = tf.Variable(FLAGS.learning_rate, trainable=False)
```

Comment:
This should be using `tfv1.get_variable('learning_rate', initializer=FLAGS.learning_rate, trainable=False)`

Reply:
Also, you can create this variable in the caller and pass it to `create_optimizer`, it's cleaner than returning multiple values here.